free-3.md

```diff
@@ -10,59 +10,116 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+This idea has to do with asynchronous tasks. It's not super well fleshed out,
+but I know it will be something similar to cron, but making it very
+easy to automate tasks.
 
-### Why a language?
-_Why is a DSL appropriate for your user(s)? How does it address the need?_
-
+I've never used cron myself (I would definitely try it out before I try to
+make a new version or anything), but I've heard that it can be a bit
+clunky and not super intuitive to learn.  Hopefully anyone who needs to
+fire off tasks to run at intervals would be able to use a more friendly
+software to do so.
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
-
+I would be excited for the results of the language, being able to just
+write a quick description of what to run, how, and when, and have it
+just run when it should.  I would also be excited to learn a bit more
+about how cron works and the best way to abstract cron functionality
+into abstract language concepts.
```

Comment:
I wonder if providing some sort of GUI might be a way to make this happen more easily than a language...